src/FeApp.js

```diff
@@ -62,6 +63,7 @@ export class FeApp extends LitElement {
     return html`
       <main>
         <h1>${this.title}</h1>
+        <fe-card></fe-card>
```

Comment:
we dont need Card directly in application

Reply:
yes removed it